package.json

```diff
@@ -35,23 +35,26 @@
     "url": "https://github.com/rangle/angular2-redux-starter/issues"
   },
   "dependencies": {
-    "@types/jasmine": "^2.2.33",
-    "@types/node": "^6.0.38",
     "http-server": "^0.9.0"
   },
   "devDependencies": {
-    "@angular/common": "2.0.0-rc.6",
-    "@angular/compiler": "2.0.0-rc.6",
-    "@angular/core": "2.0.0-rc.6",
-    "@angular/http": "2.0.0-rc.6",
-    "@angular/platform-browser": "2.0.0-rc.6",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.6",
+    "@angular/common": "2.0.0-rc.7",
+    "@angular/compiler": "2.0.0-rc.7",
+    "@angular/compiler-cli": "^0.6.1",
+    "@angular/core": "2.0.0-rc.7",
+    "@angular/http": "2.0.0-rc.7",
+    "@angular/platform-browser": "2.0.0-rc.7",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.7",
+    "@angular/platform-server": "2.0.0-rc.7",
     "@angular/router": "^3.0.0-rc.2",
+    "@types/jasmine": "^2.2.33",
+    "@types/node": "^6.0.38",
+    "@types/core-js": "^0.9.32",
     "autoprefixer": "^6.3.3",
     "awesome-typescript-loader": "^2.2.1",
     "babel-core": "^6.9.1",
```

Comment:
For the build and dev, removing babel-xxx is fine but running karma tests in this case rises problem like: 

`Failed to parse file: /Users/newAccount/Desktop/angular2-starter/src/tests.entry.ts
`

I think the reason for this comes from `istanbul-instrumenter-loader`, mainly because turning off `istanbul-instrumenter-loader` fixes the error (but we loses coverage report). It seems `istanbul-instrumenter-loader` still need babel to transpile ts?